libs/@blockprotocol/graph/src/graph-embedder-handler.ts

```diff
@@ -38,7 +38,7 @@ export class GraphEmbedderHandler
     blockEntity?: Entity;
     blockGraph?: BlockGraph;
     callbacks?: Partial<EmbedderGraphMessageCallbacks>;
-    element: HTMLElement;
+    element?: HTMLElement | null;
     entityTypes?: EntityType[];
     linkedAggregations?: LinkedAggregations;
     readonly?: boolean;
```

Comment:
`initialize` isn't called in the constructor here, regardless of whether there's an element or not, which forces people to call `initialize` even if they originally provided an element.

I believe it originally was not present because the core embedder handler doesn't do anything on initialization, but it's doing more things now so probably worth calling – or moving the `initialize` calls into the super class so that we don't have to worry about doing it in all its implementations.